docs/qa-review-template.md

```
@@ -0,0 +1,143 @@
+# PMLE Question QA Review Template
+
+**Review Date:** _________________  
+**Reviewer:** _________________  
+**Sample Size:** _________________  
+**Questions Reviewed:** __ / __
```

markdownlint-cli2 (0.18.1)

Spaces inside emphasis markers

(MD037, no-space-in-emphasis)